plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -8,8 +8,8 @@
 from typing import List, Optional, Iterator, Tuple, Any, Dict, Deque
 
 from dataclasses import dataclass
-from functools import partial
 
+from dbt.flags import PROFILES_DIR
```

Comment:
Think you need to remove this @barrywhart 

```suggestion
```

Reply:
Done!